webapp/channels/src/components/admin_console/permission_schemes_settings/permission_team_scheme_settings/team_in_list/team_in_list.tsx

```diff
@@ -16,41 +16,39 @@ type Props = {
     isDisabled: boolean;
 }
 
-export default class TeamInList extends React.PureComponent<Props> {
-    handleRemoveTeam = () => {
-        const {team, isDisabled, onRemoveTeam} = this.props;
+const TeamInList = ({team, isDisabled, onRemoveTeam}: Props) => {
```

Comment:
Can you put each prop in a different line?
```suggestion
const TeamInList = ({
    team,
    isDisabled,
    onRemoveTeam,
}: Props) => {
```